internal/config/config_type.go

```diff
@@ -310,29 +342,7 @@ type InvalidValueError struct {
 	ValidValues []string
 }
 
-func (e InvalidValueError) Error() string {
-	return "invalid value"
-}
-
-func validateConfigEntry(key, value string) error {
-	validValues, found := configValues[key]
-	if !found {
-		return nil
-	}
-
-	for _, v := range validValues {
-		if v == value {
-			return nil
-		}
-	}
-
-	return &InvalidValueError{ValidValues: validValues}
-}
-
 func (c *fileConfig) Set(hostname, key, value string) error {
-	if err := validateConfigEntry(key, value); err != nil {
```

Comment:
This removes validation in `Set` and does not replace it with anything. Is this a regression?

Reply:
I did change the responsibility of the function here by removing the validation and putting it directly in the `config set` command. Specifically in the `setRun` function. This was the only caller of the `Set` function using user input that I could find so it felt okay to leave the responsibility of checking the validity of the value up to the `config set` command where the user input is actually coming in.